met-api/src/met_api/resources/staff_user.py

```diff
@@ -93,6 +93,30 @@ def get(user_id):
         return user, HTTPStatus.OK
 
 
+@cors_preflight('PATCH')
+@API.route('/<user_id>/status')
+class StaffUserStatus(Resource):
+    """User controller class."""
+
+    @staticmethod
+    @cross_origin(origins=allowedorigins())
+    @_jwt.has_one_of_roles([Role.TOGGLE_USER_STATUS.value])
```

Comment:
TOGGLE_USER_STATUS 
If i get a better name for this role , I probbly will change it 
but good for now